install.md

```diff
@@ -38,7 +38,8 @@ The following dependencies are required to build it:
 - go >= 1.13
 - go-md2man
 - meson >= 0.58.0
-- ninja
+- shadow-utils-subid-devel
+- ninja-build
```

Comment:
Nitpick: it will be better to avoid using names from the Fedora family.

Toolbx supports other distributions like Arch Linux and Ubuntu, and those can have different names for these packages.  eg., the equivalent of Fedora's `shadow-utils-subid-devel` is [libsubid-dev](https://packages.ubuntu.com/source/mantic/shadow).  It will become really difficult to cover the names from every possible distribution, so let's stick to the upstream project names.

Reply:
I took the liberty to update the branch with the upstream names.